### PR TITLE
Handle OVER in grid

### PR DESCRIPTION
**Pull request by jonnyh404**

### CODE REVIEW

File: tests/Nexus/test_nexus_file_operations.py [hunk 1]
```diff
@@ -819,3 +819,29 @@ def side_effect(filename, mode, errors=None):
     result = nfo.load_file_as_list(file_path)
     # Assert
     assert result == expected_file_as_list
+
```
Comment (kwr3) [inline tests/Nexus/test_nexus_file_operations.py:822]: good tests

File: ResSimpy/Nexus/DataModels/StructuredGrid/NexusGrid.py [hunk 2]
```diff
@@ -1423,3 +1423,66 @@ def multir(self) -> list[NexusMultir]:
...
+                    threshold_value = None
+
+            if nfo.check_token('OVER', line):
+                reading_over = True
+                over_split_line = nfo.split_line(line)
+                arrays = over_split_line[over_split_line.index('OVER')+1:]
+
+        return overs_list
+
+    def get_overs(self) -> list[NexusOver]:
```
Comment (kwr3) [inline ResSimpy/Nexus/DataModels/StructuredGrid/NexusGrid.py:1484]: wouldn't we normally have a NexusOvers class which is a list of NexusOver objs

**Review by kwr3 — APPROVED** [review]
Looks good.

File: tests/Nexus/test_nexus_grid_over.py [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+    # Assert
+    assert result == expected_result
+
+
+def test_read_nexus_over_faultnames():
+    # Arrange
+    file_content = """! comment
+    OVER TXF TYF TZF
+    FNAME fault_name_1
```
[the remark this answers is not in the document]
Comment (jonnyh404) [reply tests/Nexus/test_nexus_grid_over.py:58]: It shouldn't be possible.

**Pull request merged**